components/trezor_bridge/resources/trezor/iframe.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import 'trezor-connect/lib/iframe/iframe'
+const { setFetch } = require('trezor-link/lib/bridge/http')
+
+// Overriding window.fetch to catch network requests in the page handler.
+window.fetch = function (url, options) {
+  return new Promise(async (resolve, reject) => {
```

Comment:
This is where you can exclude regular relative requests, e.g. `/data/messages` or `/another.webpack.chunk.js` so that we can reduce this patch to only proxy requests to the trezor service.

```ts
if (url.startsWith('/') {
  return nativeWindowFetch(url, options)
}
```

Reply:
doesnt work. discussed in slack